src/shared/navbar/login/Login.tsx

```diff
@@ -0,0 +1,18 @@
+import AuthOverlay from "@shared/authOverlay/AuthOverlay";
+import { NavElem } from "@shared/navbar/NavElem";
+import { FC, useCallback, useState } from "react";
+
+export const Login: FC = () => {
+  const [showAuthOverlay, setShowAuthOverlay] = useState(false);
+  const handleClose = useCallback(() => {
+    setShowAuthOverlay(false);
+  }, []);
+  return (
+    <>
+      <NavElem to="#" onClick={() => setShowAuthOverlay(true)}> Login </NavElem>
```

Comment:
Not sure if the `to="#"`  here is required, is React complaining if you omit it?